stage1/usr_from_kvm/kernel/patches/0001-reboot.patch

```diff
@@ -0,0 +1,22 @@
+--- linux-4.0/kernel/reboot.c~	2015-04-12 18:12:50.000000000 -0400
++++ linux-4.0/kernel/reboot.c	2015-05-06 09:45:56.991894271 -0400
+@@ -309,7 +309,7 @@
+ 	 * halt when pm_power_off is not set do it the easy way.
+ 	 */
+ 	if ((cmd == LINUX_REBOOT_CMD_POWER_OFF) && !pm_power_off)
+-		cmd = LINUX_REBOOT_CMD_HALT;
```

Comment:
Just above the lines you change in kernel. So they would be a part of this patch.

If that were a git-patch, where there is a section for explaining the changes, I wouldn't require such comment.